CHANGES.rst

```diff
@@ -138,6 +138,12 @@ astropy.time
 astropy.units
 ^^^^^^^^^^^^^
 
+- To simplify fast creation of ``Quantity`` instances from arrays, one can now
```

Comment:
It's probably worth mentioning also in the API changes section that any previous use of ``|`` assuming it was "or" will be broken.  There's probably no sensible use of that anyway (and maybe it doesn't even work?), but it's probably better safe than sorry.

Reply:
Ignore this.  I just looked now and indeed ``|`` didn't work at all before.